src/__tests__/__snapshots__/BreadcrumbItem.js.snap

```diff
@@ -314,6 +308,12 @@ exports[`Breadcrumb.Item renders the given "as" prop 1`] = `
         "primaryShadow": "0px 1px 0px rgba(20, 70, 32, 0.1), inset 0px 2px 0px rgba(255, 255, 255, 0.03)",
         "small": "0 1px 1px rgba(27, 31, 35, 0.1)",
       },
+      "sizes": Object {
```

Comment:
We should figure out how to remove the theme object from all of our snapshot tests 😂

Reply:
i knoooooow 😅